### PR TITLE
Deploy test app to heroku

### PR DESCRIPTION
**Pull request by farridav**
- Setup github action for deploying to heroku (Secrets in place)
- Reset all data nightly at 12pm
- Move test data into fixture
- cleans up delete confirmation page a bit
delete_confirmation
- Redirect to admin on / within test app

Fixes: https://github.com/farridav/django-jazzmin/issues/38
Fixes: https://github.com/farridav/django-jazzmin/issues/31

### CODE REVIEW

File: tests/test_app/polls/middleware.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+class CleanupMiddleware(MiddlewareMixin):
+
+    @transaction.atomic
+    def process_response(self, request, response):
+        """
+        10% of requests will cause all data to be reset
+        """
+
+        if settings.PERIODIC_RESET:
+            if random.randint(1, 10) == 1:
```
Comment (TysonRV) [inline tests/test_app/polls/middleware.py:20]: I think this is a bit too often... and also a bit too random 😄 

I would rather do this on a nightly cronjob, so people can go crazy with testing if they want to, without having to worry about all data magically resetting... What do you think?

File: tests/test_app/settings.py [hunk 2]
```diff
@@ -176,11 +172,12 @@
...
-        'debug_toolbar',
-        'django_extensions',
-    ])
+    INSTALLED_APPS.extend(['debug_toolbar', 'django_extensions'])
     MIDDLEWARE.insert(0, "debug_toolbar.middleware.DebugToolbarMiddleware")
     DEBUG_TOOLBAR_CONFIG = {"SHOW_TOOLBAR_CALLBACK": lambda _: True}
+
+if not DEBUG and not TEST:
+    MIDDLEWARE.insert(1, 'django.middleware.security.SecurityMiddleware')
+    STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
```
Comment (TysonRV) [inline tests/test_app/settings.py:183]: You don't have whitenoise installed yet

File: README.md [hunk 3]
```diff
@@ -59,3 +59,11 @@ project were possible, but this project is taking a slightly different direction
 - Based on AdminLTE 3: https://adminlte.io/
 - Using Bootstrap 4: https://getbootstrap.com/
 - Using Font Awesome 5: https://fontawesome.com/
+
+
+
+TODO:
+    - whitenoise in test app, and gunicorn and postgres
+    - add cron for reset
+    - keep it clean
```
Comment (TysonRV) [inline README.md:68]: Is this still a TODO? I just saw you got them installed already. Can't see `DATABASE_URL` but I guess it's postgres?

Comment (farridav) [reply README.md]: on the heroku app its set, yes it is postgres :)

Ive updated the docs with the link to the app

File: tests/test_app/polls/middleware.py (hunk 1 above)
Comment (farridav) [reply tests/test_app/polls/middleware.py]: Noted, Ive removed this, and moved it to a nightly reset as recommended, we can adjust the frequency as necessary

File: README.md (hunk 3 above)
Comment (TysonRV) [reply README.md]: Cool beans!

**Comment by TysonRV** [comment]
Great work here! 👍

**Pull request merged**